airdrop/config.py

```diff
@@ -69,3 +70,9 @@ class DepositDetails:
     policy_id = "6f1a1f0c7ccf632cc9ff4b79687ed13ffe5b624cce288b364ebdce50"
     asset_name = "41474958"
     amount_in_lovelace = "2000000"
+
+
+class TokenTransferCardanoService:
+    url = "http://127.0.0.1:5005/cardano/AGIX/transfer"
```

Comment:
the URL should not have AGIX as hardcoded path params because that could potentially change to Nunet as well